docs/user/dashboard/edit-dashboards.asciidoc

```diff
@@ -81,6 +81,20 @@ Put the dashboard in *Edit* mode, then use the following options:
 * To delete, open the panel menu, then select *Delete from dashboard*. When you delete a panel from the dashboard, the 
 visualization or saved search from the panel is still available in Kibana.
 
+[float]
+[[sync-colors]]
+=== Synchronize colors
+
+By default, Kibana attempts to keep colors for series consistent across panels, making sure the same series name always uses the same color.
+
+In some cases this can lead to lots of different colors in a single dashboard which can be undesirable. It is possible to disable this behavior:
```

Comment:
The color synchronizing logic can make the dashboard less readable when there are too many unique series names.  It is possible to disable the synchronization behavior: